_posts/2017-07-25-editor.md

```diff
@@ -19,18 +19,16 @@ I started this blog in May of last year. I don't mean to brag, but by last April
 
 The size of my audience changed a bit in May 2017. The sharp-eyed reader may be able to spot the subtle shift in my traffic graph:
 
-{% capture fig_img %}
-[![Blog traffic graphs](/images/2017-07-25-editor/full-traffic-graph-sm.png){: .align-center}](/images/2017-07-25-editor/full-traffic-graph.png)
-{% endcapture %}
-
-{% capture fig_caption %}
-Number of unique readers visiting [mtlynch.io](https://mtlynch.io) per week
-{% endcapture %}
-
-<figure>
-  {{ fig_img | markdownify | remove: "<p>" | remove: "</p>" }}
-  <figcaption>{{ fig_caption | markdownify | remove: "<p>" | remove: "</p>" }}</figcaption>
-</figure>
+{% assign fig_caption = "Number of unique readers visiting [mtlynch.io](https://mtlynch.io) per week" | markdownify | remove: "<p>" | remove: "</p>" %}
+
+{% responsive_image_block %}
+    path: images/2017-07-25-editor/full-traffic-graph.png
```

Comment:
Unfortunately, the plugin needs the full path of the image as it uses this to build the resized images.  Therefore, dynamically setting this in the responsive-image plugin template is not possible.

However, what I can offer as a solution and what I checked in under f75ceb6 is to create a "prequel" type of include located at `_includes/image.html`.  Having a "prequel" type of include will allow you to just write `path: foo.png` within a post.  However, instead of directly calling the responsive image tags within a post, you will have to call the "prequel" include.  See below or in the `_posts/2017-7-25-editor.md` for an example:

## Example of calling the "prequel" image template in post:
{% include image.html file="foo.png" alt="foo alt" img_link=true fig_caption="foo fig caption" media_rendition="article" %}

Hope that makes sense and I'll update the PR description with this change as well.  If you don't like this as a solution I'll remove it, but not sure how else to accomplish the ability to infer automatically the image path.